src/synthesizer/particle/blackholes.py

```diff
@@ -116,26 +119,41 @@ def __init__(
             nparticles=len(masses),
         )
 
-        # Accretion rate in Msun / yr
-        self.accretion_rate = accretion_rate
-
-        # Bolometric luminosity
-        self.bol_luminosity = None
-
-        # Calculate the big bump temperature
-        self.bb_temperature = (
-            2.24e9 * self._accretion_rate ** (1 / 4) * self._masses**-0.5
+        # for the time being this takes plural quantities but gives 
+        # BlackholesComponent sing
```

Comment:
Typo in comment?

Reply:
The singular/plural issue here is fine. Just a necessary evil of the disconnect between particle and parametric but it's not front facing